src/averageAges.js

```diff
@@ -14,12 +14,18 @@
  *
  * @return {number}
  */
-function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+function calculateMenAverageAge(people, century = true) {
+  const filterPeoples = people.filter(person =>
+    person.sex === 'm'
+    && (century === Math.ceil(person.died / 100) || century === true)
```

Comment:
I'm not sure it's working correct. Try this plz
```suggestion
    && (!century &&  century === Math.ceil(person.died / 100) )
```